SPECS/libtiff/libtiff.spec

```diff
@@ -67,6 +75,17 @@ make %{?_smp_mflags} -k check
 %{_mandir}/man3/*
 
 %changelog
+* Fri Nov 18 2022 Sam Meluch <sammeluch@microsoft.com - 4.4.0-6
```

Comment:
4.4.0-7 ?

Reply:
4.4.0-6 was never committed to main, condense changelog comments instead of incrementing the release number.